spec/views/profile/index.html.slim_spec.rb

```diff
@@ -20,7 +20,8 @@
       expect(rendered).not_to have_xpath("//input[@value='Disable']")
     end
 
-    it 'contains link to delete account' do
+    xit 'contains link to delete account' do
+      pending 'temporarily disabled until we figure out the MBUN to SSN mapping'
```

Comment:
Those are the docs for version 2.0. We are using 3.5:
https://relishapp.com/rspec/rspec-core/v/3-5/docs/pending-and-skipped-examples/pending-examples

You can also try running the tests for this branch on your machine and see what I mean 😄